### ISSUE TITLE
should be possible to see revision history of Community and Business Group Process (CGBP)

### ISSUE THREAD

**Issue by dbaron** [w3c/cg-council#3]
I don't see a way to see the revision history of the [Community and Business Group Process](https://www.w3.org/community/about/agreements/) or the associated agreements.  It's possible that they've never been revised, but if/when they are revised, it seems like the revision history should be visible (via dated versions or something similar), since it seems like knowing what process was in effect at a certain time in the past might be relevant (e.g., in terms of IPR commitments).

**Comment by ianbjacobs** [comment]
Hi David,

I agree there should be a way to see revisions. Let's fix that. 

Right now there's nothing to see, however. The CG process --including the CLA-- has remained essentially unchanged  since we launched it in August 2011. We did deprecate the Community Forum in 2015, which wasn't being used. We've added a note to that effect [1] to the documentation.

Most of the work since 2011 has been around support: operational documentation (e.g., the FAQ),  charter template [2], site redesign for CGs, re-implementation by the systems team of tools used to join groups, etc.

Years ago the Community Council (and others) had discussions about changes. For example:

* Changes contemplated in 2012:
  https://www.w3.org/community/council/wiki/Cg_2012

* Changes contemplated in 2013:
  https://www.w3.org/community/council/wiki/Cg_2013

I'd be happy to discuss any of those with you.

Ian

[1] https://www.w3.org/community/about/agreements/
[2] https://w3c.github.io/cg-charter/CGCharter.html
